### PR TITLE
ISPN-15402 JGroups Raft 1.0.12.Final

### PR DESCRIPTION
**Pull request by jabolina**
https://issues.redhat.com/browse/ISPN-15402

Supersedes #11604.

Bump org.jgroups:jgroups-raft from 1.0.11.Final to 1.0.12.Final

Bumps [org.jgroups:jgroups-raft](https://github.com/jgroups-extras/jgroups-raft) from 1.0.11.Final to 1.0.12.Final.
- [Commits](https://github.com/jgroups-extras/jgroups-raft/compare/jgroups-raft-1.0.11.Final...jgroups-raft-1.0.12.Final)

---
updated-dependencies:
- dependency-name: org.jgroups:jgroups-raft dependency-type: direct:production update-type: version-update:semver-patch ...

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -343,7 +343,7 @@
       <versionx.org.jgroups.aws.jgroups-aws>2.0.1.Final</versionx.org.jgroups.aws.jgroups-aws>
       <versionx.org.jgroups.azure.jgroups-azure>2.0.2.Final</versionx.org.jgroups.azure.jgroups-azure>
       <versionx.org.jgroups.jgroups>${version.jgroups}</versionx.org.jgroups.jgroups>
-      <versionx.org.jgroups.jgroups-raft>${version.jgroups.raft}</versionx.org.jgroups.jgroups-raft>
+      <versionx.org.jgroups.jgroups-raft>1.0.12.Final</versionx.org.jgroups.jgroups-raft>
```
Comment (tristantarrant) [inline pom.xml:346]: Hm, shouldn't this change apply to `build/configuraiton/pom.xml` instead ?

Comment (jabolina) [reply pom.xml:346]: Argh, yeah, trusting too much on dependabot. Updating.

**Comment by jabolina** [comment]
Updated.

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**